### ISSUE TITLE
NPE in NetUtil in case an interface doesn't have a broadcast address

### ISSUE THREAD

**Issue by sjka** [eclipse/smarthome#5172]
I just rain into this:

```
12:53:54.759 DEBUG o.e.s.i.r.JSONResponse$ExceptionMapper[:203] - exception during REST Handling
java.lang.NullPointerException: null
	at org.eclipse.smarthome.core.net.NetUtil.getAllBroadcastAddresses(NetUtil.java:189)
	at org.eclipse.smarthome.config.core.internal.net.NetworkConfigOptionProvider.getParameterOptions(NetworkConfigOptionProvider.java:54)
	at org.eclipse.smarthome.config.core.ConfigOptionProvider.getParameterOptions(ConfigOptionProvider.java:51)
	at org.eclipse.smarthome.config.core.ConfigDescriptionRegistry.fillFromProviders(ConfigDescriptionRegistry.java:293)
	at org.eclipse.smarthome.config.core.ConfigDescriptionRegistry.getConfigOptions(ConfigDescriptionRegistry.java:262)
	at org.eclipse.smarthome.config.core.ConfigDescriptionRegistry.getConfigDescription(ConfigDescriptionRegistry.java:189)
	at org.eclipse.smarthome.io.rest.core.internal.config.ConfigDescriptionResource.getByURI(ConfigDescriptionResource.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
```

Apparently due to some VPN configuration settings there is an interface without a broadcast address:

![image](https://user-images.githubusercontent.com/2592443/36898006-3ea5d3b8-1e19-11e8-9117-7c6f5632364e.png)

**Comment by sjka** [comment]
PS: Im not sure (as there was not error description), but maybe that's the one #5160 actually intended to fix?

**Comment by triller-telekom** [comment]
I also don't know if it is the same error as the one fixed in #5160.... But i have created #5173 to fix this one :)
